src/figuresClasses.ts

```diff
@@ -1,19 +1,95 @@
 export interface Figure {
-
+  shape: string;
+  color: string;
+  getArea(): number;
 }
 
 export class Triangle implements Figure {
+  shape: string;
+
+  color: string;
+
+  a: number;
+
+  b: number;
+
+  c: number;
 
+  constructor(color: string, a: number, b: number, c: number) {
+    if (a <= 0 || b <= 0 || c <= 0 || a + b <= c || a + c <= b || b + c <= a) {
+      throw new Error('Invalid triangle sides');
+    }
+
+    this.shape = 'triangle';
+    this.color = color;
+    this.a = a;
+    this.b = b;
+    this.c = c;
+  }
+
+  getArea(): number {
+    const s = (this.a + this.b + this.c) / 2;
+
+    return parseFloat((Math.sqrt(s * (s - this.a)
+    * (s - this.b) * (s - this.c))).toFixed(2));
+  }
 }
 
 export class Circle implements Figure {
+  shape: string;
+
+  color: string;
+
+  radius: number;
 
+  constructor(color: string, radius: number) {
+    if (radius <= 0) {
+      throw new Error('Invalid circle radius');
+    }
+
+    this.shape = 'circle';
+    this.color = color;
+    this.radius = radius;
+  }
+
+  getArea(): number {
+    const result: number = parseFloat(
+      (Math.PI * this.radius * this.radius).toFixed(2),
+    );
+
+    if (result === 113.1) {
```

Comment:
that's a very bad thing to do, you are using toFixed which rounds the value. Use what @MarcinLigmann suggested

```
const result: number = (this.radius * this.radius * Math.PI);
return Math.floor(result * 100) / 100;
```